core/src/main/java/com/microsoft/applicationinsights/telemetry/ExceptionTelemetry.java

```diff
@@ -57,22 +58,20 @@ private ExceptionTelemetry() {
         initialize(data.getProperties());
     }
 
-    /**
-     * Initializes a new instance.
-     * @param stackSize The max stack size to report.
-     * @param exception The exception to track.
-     */
-    public ExceptionTelemetry(Throwable exception, int stackSize) {
```

Comment:
Are you removing this public constructor? In that case it might be a breaking change, though rarely anyone is relying on it.

Reply:
I've reverted this change.